pkg/api/defaults.go

```diff
@@ -561,13 +563,6 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 					if !cs.Properties.MasterProfile.IsVirtualMachineScaleSets() {
 						profile.Subnet = cs.Properties.MasterProfile.Subnet
 					}
-					if cs.Properties.OrchestratorProfile.OrchestratorType == Kubernetes {
-						if !cs.Properties.MasterProfile.IsVirtualMachineScaleSets() {
```

Comment:
Did this `profile.Subnet =` logic get moved elsewhere, or was it not needed?

Reply:
Right, this was just crazy code. L563-L565 above is identictal to what is removed.